include/lsst/daf/base/DateTime.h

```diff
@@ -63,15 +63,15 @@ class DateTime {
     explicit DateTime(double date, DateSystem system = MJD, Timescale scale = TAI);
     DateTime(int year, int month, int day, int hr, int min, int sec,
              Timescale scale = TAI);
-    explicit DateTime(std::string const& iso8601);
+    explicit DateTime(std::string const& iso8601, Timescale scale);
```

Comment:
The only choices DateTime supports are TAI, TT and UTC. TT is just an offset from TAI. So I think it makes sense.